### DISCUSSION TITLE
Add a photo thumbanil as a presentation of the new feed!

### DISCUSSION THREAD

**Discussion by miguelgargallo** [community/community#18478] [Other Feature Feedback, Questions, & Ideas] [answered]
Add a photo thumbanil as a presentation of the new feed!
A square photo and the content next to it.
Providing a Tweet section too to get coverage!
And polls!

**Comment by siminapasat** [comment, marked as answer]
> Add a photo thumbanil as a presentation of the new feed!

Thank you for your suggestion! Can you include a bit more details what are you thinking?

> Providing a Tweet section too to get coverage!

Not sure what do you mean - to show Tweets on GitHub, or to add the ability to share a feed item via Twitter?

> And polls!

You can currently have Polls via GitHub discussions. I'm curious how you envision using polls into your feed vs in a discussion?

Thanks for your thoughts! ✨

**Reply by miguelgargallo** [reply]
Use twitter and GitHub to growth hawking marketing the social network and attract more devs using a newsfeeds of twitter devs!

**Comment by kaushikravuri** [comment]
Add a photo thumbanil as a presentation of the new feed!
Providing a Tweet section too to get coverage!
And polls!

**Comment by DHANANJAYKUMAR1432** [comment]
Mera gfpgan login nahi ho raha hai
